server/src/main/java/org/opensearch/action/admin/indices/cache/clear/ClearIndicesCacheRequest.java

```diff
@@ -120,8 +147,23 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeBoolean(fieldDataCache);
         out.writeStringArrayNullable(fields);
         out.writeBoolean(requestCache);
+        if (out.getVersion().onOrAfter(Version.V_2_12_0)) {
+            out.writeBoolean(requestCacheOnDisk);
+        }
+        if (out.getVersion().onOrAfter(Version.V_2_12_0)) {
+            out.writeBoolean(requestCacheOnHeap);
+        }
```

Comment:
Change this to V_3_0 for now, we will later backport to 2.x. As otherwise this will break BWC tests.